tests/utils.go

```diff
@@ -1437,6 +1437,17 @@ func DeletePV(os string) {
 	}
 }
 
+func DeleteVMI(vmi *v1.VirtualMachineInstance) {
+	if vmi != nil {
+		virtCli, err := kubecli.GetKubevirtClient()
+		PanicOnError(err)
+		_, err = virtCli.VirtualMachineInstance(NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+		Expect(And(HaveOccurred(), WithTransform(errors.IsNotFound, BeFalse())), "should successfully get VMI before deletion")
```

Comment:
I don't understand this line, it's not really readable (usually you can read the gomega like text, e.g. `Expect(err).ToNot(HaveOccured)`. Can you change it to something more readable?

Reply:
It combines two assertions into one. What you find problematic?
All rights reserved to @phoracek :+1: 

I do like it.

Reply:
Ohh... but I think this is not correct syntax anyway.

Reply:
Actually This is whole thing is redundant since isNotFound  should be false, meaning no error should return.
in that case a simple Expect(err).ToNot(HaveOccured) should be good enough.
Thanks. Changed.

Reply:
It should be:
```
Expect(err).To(And(HaveOccurred(), WithTransform(errors.IsNotFound, BeFalse()))
```

Reply:
Ha, it should be BeTrue()! Sorry

Reply:
@alonSadan but then you don't know if it failed on NOT FOUND or on UNABLE TO CONNECT TO API.

Reply:
I think it will print the error when it fails, doesn't it?

Reply:
My concern is not what it prints on error but that we check for `WaitForVirtualMachineToDisappearWithTimeout`. Let's say we are waiting for it to be gone and then the client call fails for a random issue. Then we would exit as if everything was fine, but that is not the case.

Reply:
I can't tell what is this thread commenting on, but if it is on DeleteVMI alone, we can indeed check only for any error. Sorry for the confusion